heudiconv/convert.py

```diff
@@ -523,6 +523,35 @@ def update_uncombined_name(
     return filename
 
 
+def update_multiorient_name(
+    metadata: dict[str, Any],
+    filename: str,
+) -> str:
+    if "acq-" in filename:
```

Comment:
```suggestion
    if "_acq-" in filename:
```

so doesn't catch some possible (now or in the future) `megaacq-`?